examples/bond/assets/bond-dapp-stateful.py

```diff
@@ -0,0 +1,209 @@
+from pyteal import *
+
+def approval_program():
+    """
+    A stateful smart contract which will store the bond parameters:
+    issue price, nominal price, maturity date and coupon value, total amount.
+    Additionally, the App will store a reference to the currently
+    tradeable bond ASA and the current epoch number.
+    """
+
+    store_manager = Bytes("store_manager")
+    issue_price = Bytes("issue_price")
+    nominal_price = Bytes("nominal_price")
+    maturity_date = Bytes("maturity_date")
+    coupon_value = Bytes("coupon_value")
+    issuer_address = Bytes("issuer_address")
+    epoch = Bytes("epoch")
+    total = Bytes("total")
+    current_bond = Bytes("current_bond")
+    max_amount = Bytes("max_amount")
+    creator = Bytes("creator")
```

Comment:
Does this represent ASC creator? If yes, can we use `Global.CreatorAddress`?

Reply:
this is ASA_creator, will change it to `asa_creator` or `bond_token_creator`. what do you think ?